firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -3417,7 +3442,15 @@ extension BrowserViewController: TabManagerDelegate {
         }
 
         if let readerMode = selected?.getContentScript(name: ReaderMode.name()) as? ReaderMode {
-            if !isToolbarRefactorEnabled {
+            if isToolbarRefactorEnabled {
```

Comment:
An almost identical block is called 4 times in the various BrowserViewController files. I think we should have a method for this instead of duplicating it so many times.